### PR TITLE
Update webdriverio.multiremote.chat.js

### PR DESCRIPTION
**Pull request by snackycracky**
- the socket.io url is outdated. 
- also the this.timeout = 9999 is not supported anymore by mocha

### CODE REVIEW

File: examples/multiremote/webdriverio.multiremote.chat.js [hunk 1]
```diff
@@ -17,16 +17,14 @@ var WebdriverIO = require('../../build'),
...
-
     it('should open chat application', function() {
-        return matrix.init().url('http://chat.socket.io/');
+        return matrix.init().url('https://socketio-chat.now.sh/');
     });
 
     it('should login the browser', function(done) {
         browserA.setValue('.usernameInput', 'Browser A');
         browserB.setValue('.usernameInput', 'Browser B');
-        return matrix.sync().call(done);
```
Comment (snackycracky) [inline examples/multiremote/webdriverio.multiremote.chat.js]: this was throwing an error. `Either return or call done`

File: examples/multiremote/webdriverio.multiremote.chat.js [hunk 2]
```diff
@@ -17,16 +17,14 @@ var WebdriverIO = require('../../build'),
 
 describe('multiremote example', function() {
 
-    this.timeout = 99999999;
-
     it('should open chat application', function() {
-        return matrix.init().url('http://chat.socket.io/');
+        return matrix.init().url('https://socketio-chat.now.sh/');
```
Comment (snackycracky) [inline examples/multiremote/webdriverio.multiremote.chat.js]: this is the chat-iframe on the official socket.io chat-demo site.

File: examples/multiremote/webdriverio.multiremote.chat.js (hunk 1 above)
Comment (christian-bromann) [inline examples/multiremote/webdriverio.multiremote.chat.js]: @snackycracky let's rather return the promise

Comment (snackycracky) [inline examples/multiremote/webdriverio.multiremote.chat.js]: the only thing that would work is:
```
... 
 matrix.sync().call(done);
 return Promise.resolve();
```
it's a little cleaner

File: examples/multiremote/webdriverio.multiremote.chat.js [hunk 3]
```diff
@@ -17,16 +17,15 @@ var WebdriverIO = require('../../build'),
...
     });
 
     it('should login the browser', function(done) {
         browserA.setValue('.usernameInput', 'Browser A');
         browserB.setValue('.usernameInput', 'Browser B');
-        return matrix.sync().call(done);
+        matrix.sync().call(done);
+        return Promise.resolve();
```
Comment (snackycracky) [inline examples/multiremote/webdriverio.multiremote.chat.js]: here is the return statement.

**Comment by snackycracky** [comment]
As I said, that’s not working, it just halts, and won’t ever continue. I am not sure why.

**Comment by christian-bromann** [comment]
@snackycracky this is a bug then, let's remove `matrix.sync()` then and merge this PR

**Comment by snackycracky** [comment]
It's not working, I tried multiple things. Just 
```
matrix.sync().call(done); // without return
```
or 
```
matrix.sync().call(done);
return Promise.resolve()
```
is actually passing the tests...

Completelt removing `matric.sync();`
or doing like this is not working:
```
return Promise.resolve().then(() => {
            return browserA.setValue('.usernameInput', 'Browser A'); 
        }).then(() => {
            return browserB.setValue('.usernameInput', 'Browser B'); 
        }).then(done)
 
```

**Comment by christian-bromann** [comment]
Then there is a bug in there. Let's remove that change then and create a different issue for this.

**Comment by snackycracky** [comment]
Should I remove 'return Promise.resolve()'? Or what change do you mean ? I would rather keep the change in favor for working examples and create an issue. Its nice when you clone a repo and the included examples are actually running.

**Comment by christian-bromann** [comment]
> Should I remove 'return Promise.resolve()'?

Yes!

> I would rather keep the change in favor for working examples and create an issue.

To make it work, just remove the `.call(done)` and the `done` parameter in the function head. The `Promise.resolve()` is just confusing adds no functionality.

**Comment by snackycracky** [comment]
ok done. Works. :shipit:

**Comment by christian-bromann** [comment]
@snackycracky so this works how it is right now? If so there is no bug then 😃

**Pull request merged**